commit 5c662ad07ab4e937323f04b5e910a595c0eeca87 (refs/changes/08/2808/1)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2017-10-18T11:27:22+02:00 (2 years ago)
    
    SAPI-293 moray storage layer's putObject implementation doesn't handle errors correctly

diff --git a/lib/server/stor/moray.js b/lib/server/stor/moray.js
index aaded7d..cb26743 100644
--- a/lib/server/stor/moray.js
+++ b/lib/server/stor/moray.js
@@ -356,6 +356,7 @@ function putObject(bucket, uuid, obj, opts, cb) {
         if (err) {
             log.error(err, 'failed to put object %s', uuid);
             cb(err);
+            return;
         }
 
         var record = {};
